app/src/pages/inside/manualLaunchesPage/manualLaunchesList/hooks/useManualLaunchesTableData/useManualLaunchesTableData.tsx

```diff
@@ -47,56 +51,127 @@ export const useManualLaunchesTableData = (data: ManualTestCase[]): RowData[] =>
           skippedTests,
           testsToRun,
           successTests,
-        }) => ({
-          id,
-          count: {
-            content: count,
-            component: (
-              <CountTag count={count} className={cx('manual-launches-list-table-cell-count')} />
-            ),
-          },
-          name,
-          startTime: {
-            content: startTime,
-            component: startTime ? (
-              <AbsRelTime
-                startTime={startTime}
-                customClass={cx('manual-launches-list-table-cell-time')}
-              />
-            ) : (
-              <span aria-label={formatMessage(COMMON_LOCALE_KEYS.NOT_APPLICABLE)}>
-                {formatMessage(COMMON_LOCALE_KEYS.NOT_APPLICABLE)}
-              </span>
-            ),
-          },
-          totalTests,
-          testRunStatus: {
-            content: testsToRun,
-            component: (
-              <SegmentedStatusBar
-                data={[
-                  { status: SegmentStatus.Passed, value: successTests ?? 0 },
-                  { status: SegmentStatus.Failed, value: failedTests ?? 0 },
-                  { status: SegmentStatus.Skipped, value: skippedTests ?? 0 },
-                ]}
-                className={cx('manual-launches-list-table-cell-status')}
-              />
-            ),
-          },
-          failedTests: {
-            content: failedTests,
-            component: (
-              <span className={cx('manual-launches-list-table-cell-failed-tests')}>
-                {failedTests}
-              </span>
-            ),
-          },
-          testsToRun: {
-            content: testsToRun,
-            component: <TestRunButton count={testsToRun} />,
-          },
-        }),
+        }) => {
+          const handleRowClick = () => setSelectedLaunchId(id);
+          const isSelected = id === selectedLaunchId;
+          const handleKeyDown = (e: React.KeyboardEvent) => {
+            if (e.key === 'Enter' || e.key === ' ') {
```

Comment:
Please create shared util for this check, we have this check already in couple places in the app